### DISCUSSION TITLE
다크/라이트 모드에 따른 본문 글 색상 변환

### DISCUSSION THREAD

**Discussion by jinny-l** [pronist/hELLO#119] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows10
- 브라우저: chrome
- 스킨 버전: 3.6.1

상우님 블로그 들어가서 확인했을 때, 다크/라이트 모드에 따라 본문 글 색상이 별도로 지정되었을 경우 자동 변환이 되었는데, 제 블로그에서는 작동하지 않고 있습니다. 혹 해결 방법이 있을까요? 

예를 들면 상우님 블로그에서 라이트 모드의 핑크색 폰트는 다크 모드에서 파란색으로 변환되고 있는 것을 확인하였습니다.

**Comment by pronist** [comment, marked as answer]
@jinny-l 버그 신고 양식에 명시된대로 링크 및 스크린샷을 남겨주세요. 그렇지 않으면 확인할 방법이 없습니다. 또한 게시글 내에서 HTML 로 색상이 강제로 변환된 경우 모드를 전환해도 강제로 바꾸지 않습니다.

#112 참고 부탁드립니다.

**Reply by jinny-l** [reply]
@pronist 스크린샷 누락하여 죄송합니다. 
다시 확인해 보니, 색상 자동 변환되는 텍스는 하이퍼링크가 삽인된 텍스트였습니다.
HTML로 색상이 변환된 경우 라이트/다크 모드에 따른 자동 색상 변환이 없는 점 확인하였습니다.
이슈 확인해주셔서 감사합니다!
